test/secrets_test.rb

```diff
@@ -91,4 +97,22 @@ def process
       assert_raises(RuntimeError) { process }
     end
   end
+
+  describe "#waiting script" do
+    let(:cmd) { Bundler.root.join("bin/wait_for_it.sh") }
+    it "times out" do
```

Comment:
`it 'times out when .done is never created'`
`it 'succeeds when .done is found'`